### ISSUE TITLE
PI v 8.0 Becomes unusable a few minutes after boot-up

### ISSUE THREAD

**Issue by DBASQLGuru** [FalconChristmas/fpp#1967]
FPP v8.0 on RPI 5, 2 TB SSD disk, cooling fan, SD Card cloned over to NVME M.2 card by FPP, static IP assigned 192.168.1.4, Remote Falcon Plug-in installed.

Behavior: Boot up okay, all menus available, FPP can play a playlist, no issues.  After a few minutes the webpage becomes unavailable. The PI shows a green light, but the interface keeps disappearing, To restore access to the dashboard / index.php - I have to do a hard re-boot of the PI.  After bootup, the page becomes available again.

I have full logging of the system and would like to submit for an RCA on why the PI is unstable after a few minutes of operation.

**Expected behavior**: 
The FPP Player should remain accessible after many hours of operation after boot up.

[FPP_Logs_20240909.1155.zip](https://github.com/user-attachments/files/16929974/FPP_Logs_20240909.1155.zip)

**Additional Attachments**
Please include the log files. To create the file:
1. Open the FPP UI
2. Open the the File Manager (Content Setup->File Manager), 
3. Select the "Logs" tab
4. Click "zip" button.
5. Attach to this ticket.

**Comment by darylc** [comment]
@DBASQLGuru

1) The syslog file in the zip is incomplete, we need to see from the time before that file started as well.  Is there are syslog.1 or similar in /var/log?
2) How are you powering this pi, is it via the official 5A PSU?
3) How is it connected to the network, I see lots of wifi dropouts and problems.

**Comment by DBASQLGuru** [comment]
@darylc : Good call on the power issue. I unplugged the SoundBlaster and the unit stayed online all day. Then I went into the WiFi and removed all entries for that interface, plugged the Sound Blaster back in, rebooted, the unit has been online for over an hour now.  Since I have no need for the WiFi this year, it's not a loss and the Show will play for the season. Is there a way to monitor the power draw inside the FPP Interface?  I'm sure that I'm not going to be the only one to have this problem now that NVME is coming to RPI 5.  Thanks for the hint on where to look at for this issue.
